jwala-services/src/main/java/com/cerner/jwala/service/jvm/impl/JvmServiceImpl.java

```diff
@@ -281,23 +281,26 @@ public void deleteJvm(final Identifier<Jvm> id, boolean hardDelete, final User u
         if (hardDelete) {
             LOGGER.info("Deleting JVM service {}", jvm.getJvmName());
 
-            if (!JvmState.JVM_STOPPED.equals(jvm.getState()) && !JvmState.FORCED_STOPPED.equals(jvm.getState())) {
+            if (!JvmState.JVM_NEW.equals(jvm.getState()) && !JvmState.JVM_STOPPED.equals(jvm.getState()) &&
+                    !JvmState.FORCED_STOPPED.equals(jvm.getState())) {
                 final String msg = MessageFormat.format("Please stop JVM {0} first before attempting to delete it",

```

Comment:
I see your point but for me it means the same thing. Anyways, let's just have the team vote on it.